environment.yml

```diff
@@ -7,10 +7,10 @@ dependencies:
 - mkl
 - numpy
 - scipy
-- matplotlib
+- matplotlib<=3.0
 - cython
 - pyqt>=5.9
-- vtk>=8
```

Comment:
And what does this help?

Reply:
I was trying to find if it was the only one having issues with python 2.7 for in appveyor. 'Cos I can build the conda enviroment in my machine. So it was just some messing arround.

Reply:
Ahh I see. I think you might be right, so I got rid of the `matplotlib` one and we can see if AppVeyor is happy